src/classes/ConfigManager.py

```diff
@@ -93,8 +108,13 @@ def setConfig(self, parser, default=False):
             parser.set('config', 'consolidationPercentage',
                        self.consolidationPercentage)
             parser.set('config', 'shuffle', self.shuffle)
+            parser.set('config', 'cacheStockData', self.cacheStockData)
             parser.set('config', 'onlyStageTwoStocks', self.stageTwoPrompt)
             parser.set('config', 'useEMA', self.useEmaPrompt)
+
+            # delete stock data due to config change
+            deleteStockData()
```

Comment:
```suggestion
            self.deleteStockData()
```
Just a Good practice 👍🏼

Reply:
Agree..